packages/migration/src/migrations/v0002_dataPixelTo1080.ts

```diff
@@ -14,6 +14,12 @@
  * limitations under the License.
  */
 
+/**
+ * Internal dependencies
+ */
+import type { PageV0, UnionElementV0 } from '../types';
+import type { StoryV1 } from './v0001_storyDataArrayToObject';
```

Comment:
This is the main problem I see happening here. In migration 30 (just an example), you'd have to import types from 10 different previous migrations and you have to check which one has the most recent text element, page, etc. It seems like a lot of work to add a migration suddenly, because you have to track down when the involved types changed last time?

Reply:
@barklund See [this comment as well](https://github.com/GoogleForCreators/web-stories-wp/pull/12320#discussion_r975099978), we could define the new types in each migration regardless if there are changes or not, then only the latest could be used.